platform/darwin/src/MGLStyleAttributeValue_Private.hpp

```diff
@@ -17,4 +19,7 @@
 - (mbgl::style::PropertyValue<std::vector<float> >)mbgl_numberArrayPropertyValue;
 - (mbgl::style::PropertyValue<uint8_t>)mbgl_enumPropertyValue;
 
+// Convert mbgl types to darwin types
+- (id <MGLStyleAttributeValue>)mbgl_colorPropertyValueWith:(mbgl::style::PropertyValue<mbgl::Color>)color;
```

Comment:
`-mbgl_colorPropertyValueWith:` doesn’t conform to Cocoa naming conventions. How about `-mbgl_colorPropertyValueWithColor:`?